zubhub_frontend/zubhub/src/views/create_activity/script.js

```diff
@@ -673,7 +673,7 @@ export const submitForm = async ({ step1Values, step2Values, props, state, handl
     const formData = {
         ...formDataStep1,
         ...step === 2 ? step2Values : {},
-        publish: step === 2 ? true : false
+        publish: false
```

Comment:
after discussing this with @tuxology we've come to agree that it should be the staff members who publish activities after making any necessary changes. That's why we now added a whole new tab for unpublished activities for staff members to see. I assume it's because we want activities to be presented in a more monitored and controlled way than projects.